### PR TITLE
Fixed reference to modules

### PR DESCRIPTION
**Pull request by sbuliarca**

### CODE REVIEW

**Review by a-june — APPROVED** [review]

File: dev-aws/kafka-shared/customer-billing-invoice-fulfillment.tf [hunk 1]
```diff
@@ -13,7 +13,7 @@ resource "kafka_topic" "invoice_fulfillment" {
 }
 
 module "invoice_fulfillment_producer" {
-  source = "../../modules/producer"
+  source = "git@github.com:utilitywarehouse/kafka-cluster-config//modules/consumerr"
```
Comment (a-june) [inline dev-aws/kafka-shared/customer-billing-invoice-fulfillment.tf:16]: damn, just noticed `consumerr`

Comment (sbuliarca) [reply dev-aws/kafka-shared/customer-billing-invoice-fulfillment.tf:16]: oh buoy

Comment (a-june) [reply dev-aws/kafka-shared/customer-billing-invoice-fulfillment.tf:16]: sorry, was trigger happy

Comment (sbuliarca) [reply dev-aws/kafka-shared/customer-billing-invoice-fulfillment.tf:16]: https://github.com/utilitywarehouse/kafka-cluster-config/pull/51/files

Comment (sbuliarca) [reply dev-aws/kafka-shared/customer-billing-invoice-fulfillment.tf:16]: I think double `//` are expected

**Pull request merged**